main.go

```diff
@@ -329,3 +309,72 @@ func filterTimetableBySubjects(t timetable.Timetable, codes []string) timetable.
 	}
 	return filtered
 }
+
+var (
+	subjectscacheExpirationTime = time.Hour * 4
+	subjectscache 				= cache.New(subjectscacheExpirationTime, time.Hour*6)
```

Comment:
Piccola cosa, ma rinominabili entrambe in `subjectsCache...`, con la C maiuscola.